data/json/construction.json

```
@@ -3869,7 +3869,7 @@
     "description": "Build Reinforced Plastic Window",
     "//": "Step 2: Build curtain for reinforced plastic window",
     "category": "REINFORCE",
-    "required_skills": [ [ "fabrication", 9 ] ],
+    "required_skills": [ [ "fabrication", 3 ] ],
```

```suggestion
    "required_skills": [ [ "fabrication", 1 ] ],
```
If you go with my description, otherwise should be two.